content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -72,8 +72,7 @@ <h2 style="color: #3771e3;">Cluster Diagram</h2>
         <div class="row">
             <div class="col-md-8">
                 <p><b>The Master is responsible for managing the cluster.</b> The master coordinates all activities in your cluster, such as scheduling applications, maintaining applications' desired state, scaling applications, and rolling out new updates.</p>
-                <p><b>A node is a VM or a physical computer that serves as a worker machine in a Kubernetes cluster.</b> Each node has a Kubelet, which is an agent for managing the node and communicating with the Kubernetes master. The node should also have tools for handling container operations, such as Docker or rkt. A Kubernetes cluster that handles production traffic should have a minimum of three nodes.</p>
-
+                <p><b>A {{ < glossary_tooltip text="node" term_id="node" >}} is a VM or a physical machine that can run Kubernetes {{ < glossary_tooltip text="Pods" term_id="pod" >}}.</b> Each node has a {{ < glossary_tooltip text="kubelet" term_id="kubelet" >}}, which is an agent for managing the node and communicating with the Kubernetes master. A Kubernetes cluster that handles production traffic should have a minimum of three nodes.</p>
```

Comment:
I'm not sure if this will work or not in the `.html` file.  I guess the preview will tell us :p.

Reply:
I've pushed 215da08 -> 52da08d, dropping the `glossary_tooltip` bits from the `.html` files.